docs/overview/office-add-ins-fundamentals.md

```diff
@@ -88,42 +88,17 @@ For detailed information about the manifest, see [Office Add-ins XML manifest](a
 
 An Office Add-in can use the Office JavaScript APIs to interact with content in the Office document where the add-in is running. 
 
-#### Accessing the Office JavaScript library
+#### Accessing the Office JavaScript API library
```

Comment:
In fact, it *is* identical content. An include file is being used to inject the same content in both places -- and as you've suggested, the intent is to avoid context-switching of jumping back and forth between the two articles.